R/plotpercentile.R

```diff
@@ -40,31 +42,46 @@ plotpercentile <- function(x){
     aes <- ggplot2::aes
     element_blank <- ggplot2::element_blank
 
-    p<- ggplot2::ggplot(data=plotdata[plotdata$type=="Permutation", ], aes(x=x, y=y, group=label), aspect="iso") +
-       ggplot2::scale_colour_gradientn("log(p-val)",
-                                       colours=palette$grpalette,
-                                       values = palette$values,
-                                       limits = palette$limits
-       ) +
-       ggplot2::geom_path(color="grey80") +
-       ggplot2::geom_path(data=plotdata[plotdata$type=="Percentile", ], aes(x=x, y=y, group=label), color="grey40", size=1.25)+
-       ggplot2::geom_path(data=plotdata[plotdata$type=="Actual", ], aes(x=x, y=y, group=label, color = cc.10))+
-       ggplot2::geom_point(data=plotdata[plotdata$type=="Actual", ], aes(x=x, y=y, group=label, color = cc.10), size=2) +
-       ggplot2::theme_bw() +
-       ggplot2::theme(axis.title.x = element_blank(),
+    p<- ggplot2::ggplot(data=plotdata[plotdata$type=="Permutation", ], aes(x=x, y=y, group=label), aspect="iso")
+    if (coloring %in% c("p","p.adj","fish")) {
+      p <- p +
+        ggplot2::scale_colour_gradientn("log(p-val)",
+                                        colours=palette$grpalette,
+                                        values = palette$values,
+                                        limits = palette$limits
+       )
+    } else if (coloring != "Terms") {
+      p <- p +
```

Comment:
Whatever editor you are using, it's leaving trailing spaces at the end of lines. If you are using RStudio, there's an option "String trailing horizontal whitespace when saving" to enable under "Options - Code - Saving".

This is to reduce the chance of conflict due to changes in spaces (most editor nowadays have the option on by default)